.github/workflows/browser-e2e.yaml

```diff
@@ -0,0 +1,20 @@
+name: Browser E2E
```

Comment:
I copied this file from pion/webrtc. Please update it for dtls repository.

Reply:
Renamed to just `E2E`